website/addons/evernote/settings/local-dist.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
There should be default values for these defined in defaults.py to prevent AttributeErrors for developers who have not configured Evernote.